recipes/xvfbwrapper/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.2.8" %}
+
+package:
+  name: xvfbwrapper
+  version: {{ version }}
+
+source:
+  fn: xvfbwrapper-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/x/xvfbwrapper/xvfbwrapper-{{ version }}.tar.gz
+  md5: c8ce7c35fc555a6af272e7e17cf0b5c0
+
+build:
+  number: 0
+  # error: option --single-version-externally-managed not recognized
```

Comment:
Same story as `nipype` as `xvfbwrapper` seems to use `setuptools` if it is installed.